server/index.js

```diff
@@ -22,17 +22,17 @@ app.get('/app/:gameid', (req, res) => {
 
 router.get('/gamereviews/:gameid', async (req, res) => {
   let { gameid } = req.params;
-  if (parseInt(gameid) <= 0 || parseInt(gameid) > 100) {
+  console.log(req.originalUrl)
+  if (parseInt(gameid) <= 0 || parseInt(gameid) > 30000) {
     res.status(400).json({ error: 'Invalid game ID. Please use a number between 1 and 100.' });
     return;
   }
   try {
     // Get steamPurchased, otherPurchased counts via db call
     let { direct, key } = await getPurchaseTypeDataForGameId(gameid);
-
     // Get review data, sorted/filtered according to URLSearchParams
     let data = await getReviewsByGameIdWithOptions(gameid, req.query);
-
+    console.log(data.length);
```

Comment:
Cool if this is here for ongoing debugging, but probably should be commented out before this part is complete.

Reply:
Yeah, I am still in WIP, I want to utilize joined tables, that was just there to see how many reviews I am getting for different game Ids, before when I only had up to 100 game id, each game would have 30k+ reviews... and it would take forever to manipulate that amount of data. Having up to 30,000 ids for games lowered review count for each game id to approx 300~